components/service/glean/src/test/java/mozilla/components/service/glean/TimespanMetricTypeTest.kt

```diff
@@ -1,7 +1,5 @@
 package mozilla.components.service.glean
```

Comment:
GAH! Not your fault, but this is missing the license header too :'( Any chance you could add it?